flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/runtime/stream/sql/GroupWindowITCase.scala

```diff
@@ -412,10 +412,8 @@ class GroupWindowITCase(mode: StateBackendMode, useTimestampLtz: Boolean)
     tEnv.sqlQuery(sql).toAppendStream[Row].addSink(sink)
```

Comment:
nips: perhaps it's better to add some more agg calls besides `count(1)`, which happens to act as 'inputs counter' as well...